_includes/hosting.html

```diff
@@ -4,7 +4,7 @@
   <div class="host-contents">
   <div class="host-aside">
     Missed the Global Day of Coderetreat because there wasn't an event near you?
-    <a href="/hosts" target="_blank">Host your own!</a>
+    <a href="/pages/hosting/hosts" target="_blank">Host your own!</a>
```

Comment:
It's even better if you replace the href attribute with a [jekyll link](http://jekyllrb.com/docs/templates/#link) because that will give a compile time error if the linked to file does not exist